mono/mini/interp/interp.c

```diff
@@ -769,13 +768,10 @@ interp_walk_stack_with_ctx (MonoInternalStackWalk func, MonoContext *ctx, MonoUn
 			fi.il_offset = -1;
 			fi.type = FRAME_TYPE_MANAGED_TO_NATIVE;
 		} else {
-			MonoMethodHeader *hd = mono_method_get_header_checked (fi.method, &error);
-			mono_error_cleanup (&error); /* FIXME: don't swallow the error */
 			fi.type = FRAME_TYPE_MANAGED;
-			fi.il_offset = frame->ip - (const unsigned short *) hd->code;
+			fi.il_offset = frame->ip - frame->imethod->code;
```

Comment:
@lewurm Shouldn't we provide a cil offset instead of mint offset ?

Reply:
actually, we should set `native_offset` here; `il_offset` is going to be set here: https://github.com/mono/mono/blob/8593149b0be0f58a98d798b42aef10c58cdc8069/mono/mini/mini-exceptions.c#L1150 (once we unified exception handling)